packages/draft-js-export-html/typings/index.d.ts

```diff
@@ -3,9 +3,12 @@
 declare module 'draft-js-export-html' {
     import draftjs = require("draft-js");
 
+    interface MarkupGenerator {
+        renderBlockContent(block: draftjs.ContentBlock): string;
+    }
```

Comment:
Let's use type alias syntax (similar to below) and avoid "interface"

Reply:
This doesn't seem right. I think it should be:
```js
type MarkupGenerator = (block: draftjs.ContentBlock) => string;
```

Reply:
But, `MarkupGenerator` is a class, and `generator` is instance of it.
It uses like this:
`generator.renderBlockContent(block)`, [example](https://github.com/sstur/draft-js-utils/issues/33#issuecomment-392557416).

Reply:
Oh, in that case, you're right and I'm wrong. Let me take a closer look. Thanks!